docker/Dockerfile.gateway

```diff
@@ -0,0 +1,28 @@
+
+ARG spdk_version
+
+FROM spdk:${spdk_version:-latest} as spdk
+
+RUN dnf update -y \
+    && dnf install -y libaio numactl-libs python3-pip librbd-devel.x86_64 python-rados \
+    && dnf clean all \
+    && pip3 --no-cache-dir install --upgrade pip \
+    && python3 -m pip install grpcio \
+    && python3 -m pip install grpcio-tools
+
+COPY . /app/
+
+RUN ln -s /usr/local/bin/nvmf_tgt /usr/libexec/nvmf_tgt
```

Comment:
Where's the `make` invoke for the grpc proto files? Is that expected to happen outside the container? I'd suggest to keep the same approach as for the spdk: one image-stage for building the proto files, and a separate image (in the same Dockerfile) for running the gateway (without the build deps: librbd-devels, grpcio-tools).

Reply:
@epuertat will take a look. By the way, what is the ceph image I need to include in the docker file?

Reply:
@guptsanj I read that many projects usually version the generated proto files, but they still generate those at build time to ensure they are up to date.

Regarding the Ceph image, I guess @trociny and @idryomov meant [this one](https://quay.io/repository/ceph/ceph?tab=tags).